device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/platform_ndk.json

```diff
@@ -42,7 +42,7 @@
     },
     {
       "key": "sonic_log_level",
-      "stringval": "error"
+      "stringval": "debug"
```

Comment:
What is this setting to "debug" is used for?  Does this implies that platform related issues will only be logged with "debug" level setting which I think currently all syslog setting is to only capture at "Notice" level or above.  Would this change end up filter out all platform syslogs even for the one that may require interventions?  Or this will cause all platform events be logged all the time which may be too excessive?  just trying to understand the implication of this change as it was not described in your PR description...

Reply:
This setting is for Nokia NDK to log any message's loglevel is higher than and equal to debug level at beginning of the initialization.  NDK will set the log level to Warning when all hardware init is complete.  This will not impact the SONIC syslog.